package.json

```diff
@@ -11,27 +11,18 @@
     "index.js"
   ],
   "scripts": {
-    "test": "jest && npm run lint",
-    "lint": "xo index.js __test__/*.test.js"
+    "test": "node test/test.js"
```

Comment:
Any particular reason you chose this over `uvu test -i fixture` ?

Or `uvu -r esm test -i fixture` for that matter (since `esm` is already here)